### ISSUE TITLE
Refresh main page

### ISSUE THREAD

**Issue by JustFrederik** [jellyfin/Swiftfin#904]
**Describe the feature you'd like**
A main page refresh on closing the watched series and a way to refresh it manually.

**Additional context**
It would be great if the app would refetch the main page when stopping to watch a series and when the screen is pulled down on the main page.  When you start to watch a series on your device. Stop and start again the app will pick up where you last started the app. When you did something like add new movies they won't show up without restarting the app. A way to fix that would be to add a reload feature. I thought of something like this: 
[Image](https://images.macrumors.com/t/unQL9NQx5FxQ0sHXWf-0DFaVN9I=/1600x0/article-new/2021/09/ios-15-pull-to-refresh-how-to.jpg) I think it’s this [Stackoverflow](https://stackoverflow.com/questions/24475792/how-to-use-pull-to-refresh-in-swift#24476087)

**Comment by rwjack** [comment]
Yeah this has been quite annoying since I switched from Android TV Jellyfin to Apple TV Swiftfin.

Newly added content isn't shown either on Swiftfin, you have to restart the app.
The watched boolean doesn't update after finishing a movie or an episode.
All of these could potentially be fixed with a refresh after backing out from currently playing media.

**Comment by JustFrederik** [comment]
@rwjack it seems like it’s partially implemented. The reload on swipe down works if you compile it yourself. It would be great if the latest version would be added to the App Store.

**Comment by JustFrederik** [comment]
To be more specific the reload home page function needs to be added here. https://github.com/jellyfin/Swiftfin/blob/eb17300061315ff310749871f5b79032385ba55a/Swiftfin/Views/VideoPlayer/Overlays/Components/TopBarView.swift#L31
its probably very easy to implement, but i couldnt find the function that triggers the reload

**Comment by dacarlin** [comment]
Hi everyone, quick thanks for everyone’s work on Swiftfin. It’s a joy! Just wondering on the state of this issue, or if any help is needed. 

In searching for a workaround, I found that another way to trigger the refresh on tvOS with Swiftfin 1.0.1 (70) is to go to Settings (gear icon) > Switch User and then select the same user again. 

Additionally, it looks like @nokeeo created a PR to address this issue and it was closed by @LePips so just wondering if there is an existing fix in the works, or if anything is needed, or if I could help in any way
